soc/renesas/ra/ra8m1/soc.c

```diff
@@ -17,10 +17,14 @@
 #include <zephyr/arch/arm/nmi.h>
 #include <zephyr/irq.h>
 #include <zephyr/logging/log.h>
+#include <zephyr/sys/barrier.h>
+
 LOG_MODULE_REGISTER(soc, CONFIG_SOC_LOG_LEVEL);
 
 #include <bsp_api.h>
 
+#define CCR_CACHE_ENABLE (0x000E0201)
```

Comment:
Can this value be expressed in the cmsis definition?

Reply:
@soburi , I changed it into
`/* Enable Instrcution cache, Branch prediction, Loop and branch info cache */
#define CCR_CACHE_ENABLE (SCB_CCR_IC_Msk | SCB_CCR_BP_Msk | SCB_CCR_LOB_Msk)`